src/__tests__/helpers/auth-test-helpers.ts

```
@@ -0,0 +1,223 @@
+import { vi } from 'vitest'
+import type { User, Session, AuthError } from '@supabase/supabase-js'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)